tests/drivers/can/api/boards/numaker_m2l31ki.overlay

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/ {
+	chosen {
+		zephyr,canbus = &canfd0;
+	};
+};
+
+&pinctrl {
+	/* CAN TX/RX --> UNO D3/D2 */
+	canfd0_default: canfd0_default {
+		group0 {
+			pinmux = <PC5MFP_CANFD0_TXD>,
+				 <PC4MFP_CANFD0_RXD>;
+		};
+	};
+};
+
+&canfd0 {
+	bus-speed = <125000>;
+	bus-speed-data = <1000000>;
```

Comment:
```suggestion
```

Not needed anymore. There are defaults in Kconfig now.

Reply:
yes, it's the same with defaults and I removed it.